ports/lapacke/portfile.cmake

```
@@ -0,0 +1,80 @@
+include(vcpkg_find_fortran)
+SET(VCPKG_POLICY_EMPTY_INCLUDE_FOLDER enabled)
+
+set(lapacke_ver 3.8.0)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO  "Reference-LAPACK/lapack"
+    REF "v${lapacke_ver}"
+    SHA512 17786cb7306fccdc9b4a242de7f64fc261ebe6a10b6ec55f519deb4cb673cb137e8742aa5698fd2dc52f1cd56d3bd116af3f593a01dcf6770c4dcc86c50b2a7f
+    HEAD_REF master
+)
+
+set(VCPKG_CRT_LINKAGE_BACKUP ${VCPKG_CRT_LINKAGE})
+vcpkg_find_fortran(FORTRAN_CMAKE)
+if(VCPKG_USE_INTERNAL_Fortran)
+    if(VCPKG_CRT_LINKAGE_BACKUP STREQUAL static) 
+        message(FATAL_ERROR "LAPACKE cannot be used without supplying an external fortran compiler")
+    endif()
+endif()    
+vcpkg_configure_cmake(
+        PREFER_NINJA
+        SOURCE_PATH ${SOURCE_PATH}
+        OPTIONS
+            "-DUSE_OPTIMIZED_LAPACK=ON"
+            "-DUSE_OPTIMIZED_BLAS=ON"
+            "-DLAPACKE=ON"
+            ${FORTRAN_CMAKE}
+        )

```

```suggestion
vcpkg_cmake_configure(
    SOURCE_PATH ${SOURCE_PATH}
    OPTIONS
        "-DUSE_OPTIMIZED_LAPACK=ON"
        "-DUSE_OPTIMIZED_BLAS=ON"
        "-DLAPACKE=ON"
        ${FORTRAN_CMAKE}
)
```
vcpkg_configure_cmake() has been deprecated. We prefer to use the new function `vcpkg_cmake_configure()` for new ports.